components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/ActionManagementServiceImpl.java

```diff
@@ -150,12 +225,25 @@ public Action getActionByActionId(String actionType, String actionId, String ten
                 IdentityTenantUtil.getTenantId(tenantDomain));
     }
 
+    /**
+     * Update endpoint authentication of a given action.
+     *
+     * @param actionType     Action type.
+     * @param actionId       Action ID.
+     * @param authentication Authentication Information to be updated.
+     * @param tenantDomain   Tenant domain.
+     * @return Updated action.
+     * @throws ActionMgtException if an error occurred while updating endpoint authentication information.
+     */
     @Override
     public Action updateActionEndpointAuthentication(String actionType, String actionId, Authentication authentication,
                                                      String tenantDomain) throws ActionMgtException {
 
         String resolvedActionType = getActionTypeFromPath(actionType);
         Action existingAction = checkIfActionExists(resolvedActionType, actionId, tenantDomain);
+        // Validate endpoint authentication details in the request.
```

Comment:
This comment is redundant as method is self explanatory

Reply:
Updated with 449b9ab73f2e4a8c33cbd36a01a9283e1e1415e4